virtual_rainforest/models/abiotic/abiotic_model.py

```diff
@@ -122,5 +136,63 @@ def update(self, time_index: int, **kwargs: Any) -> None:
             time_index: The index of the current time step in the data object.
         """
 
+        wind_update_inputs: dict[str, DataArray] = {}
+
+        for var in ["layer_heights", "leaf_area_index", "air_temperature"]:
+            selection = (
+                self.data[var]
+                .where(self.data[var].layer_roles != "soil")
+                .dropna(dim="layers")
+            )
+            wind_update_inputs[var] = selection
+
+        wind_update = wind.calculate_wind_profile(
+            canopy_height=self.data["canopy_height"].to_numpy(),
+            wind_height_above=(self.data["canopy_height"] + 15).to_numpy(),  # TODO
+            wind_layer_heights=wind_update_inputs["layer_heights"].to_numpy(),
+            leaf_area_index=wind_update_inputs["leaf_area_index"].to_numpy(),
+            air_temperature=wind_update_inputs["air_temperature"].to_numpy(),
+            atmospheric_pressure=self.data["atmospheric_pressure"].to_numpy()[0],
+            sensible_heat_flux_topofcanopy=(
+                self.data["sensible_heat_flux_topofcanopy"].to_numpy()
+            ),
+            wind_speed_ref=(
+                self.data["wind_speed_ref"].isel(time_index=time_index).to_numpy()
+            ),
+            wind_reference_height=(self.data["canopy_height"] + 10).to_numpy(),
+            turbulence_sign=True,
+            abiotic_constants=AbioticConsts(),
+            core_constants=CoreConsts(),
```

Comment:
Doesn't this make it impossible for the user to change the constants, because default values will always be used?

Reply:
oh yes, you're right. I copied the arguments and missed changing that.